gamebook/terminal_output.py

```diff
@@ -11,3 +12,7 @@ def output(self, content):
 
     def clear(self):
         os.system("cls")
+
+    def exit(self, exit_reason):
+        self.output(exit_reason)
+        sys.exit()
```

Comment:
still, do you want to close (the server for example) each time this happens?

Reply:
no, but this one is for the local terminal version.
when we add the config we can change it right?

Reply:
ok